modules/generate/generate.go

```diff
@@ -34,6 +35,64 @@ func GetRandomString(n int) (string, error) {
 	return string(buffer), nil
 }
 
+// GetRandomPassword generates a random password
+func GetRandomPassword(n int) (string, error) {
+	const lower = "abcdefghijklmnopqrstuvwxyz"
+	const upper = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	const digit = "0123456789"
+	const spec = "!@#$%^&*()_-+=[]'\";:/?.>,<`~"
```

Comment:
I'm seeing this (unedited!):
![image](https://user-images.githubusercontent.com/18600385/63445668-2cff6780-c40f-11e9-9e72-2fbc52733ae5.png)
Is there an escape character in the string that causes the red background on `%`? (e.g. `\`). If it is, it's probably a bad idea to accept/generate a password with such a character. The same thing for quotes, at least single ones.

Reply:
You are right